editor/editor_node.cpp

```diff
@@ -4172,8 +4199,8 @@ void EditorNode::_scene_tab_script_edited(int p_tab) {
 		inspector_dock->edit_resource(script);
 }
 
-void EditorNode::_scene_tab_closed(int p_tab) {
-	current_option = SCENE_TAB_CLOSE;
+void EditorNode::_scene_tab_closed(int p_tab, int option) {
```

Comment:
This line gives me the follow errors, and at times prevents me from closing tabs.  The signal still passes  a single argument.

![image](https://user-images.githubusercontent.com/13004169/56723366-7c7ba280-6751-11e9-9184-218fcdaa41e5.png)